<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>269</width>
      <height>318</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QGraphicsView" name="graphicsView"/>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Введите имя файла</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="filename"/>
             </item>
             <item>
              <widget class="QLabel" name="Error_label">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="review">
               <property name="text">
                <string>Обзор</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="solve">
               <property name="text">
                <string>Рассчитать</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Выберите метод кластеризации</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QRadioButton" name="kmethod">
             <property name="text">
              <string>Метод k-средних</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="set_k">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
             <property name="displayIntegerBase">
              <number>10</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QRadioButton" name="single">
           <property name="text">
            <string>single</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="complete">
           <property name="text">
            <string>complete</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="average">
           <property name="text">
            <string>average</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="weighted">
           <property name="text">
            <string>weighted</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="centroid">
           <property name="text">
            <string>centroid</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="median">
           <property name="text">
            <string>median</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="ward">
           <property name="text">
            <string>ward</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="newmethod">
           <property name="text">
            <string>DBSCAN</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>806</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
